src/index.html

```diff
@@ -4,9 +4,470 @@
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="cards">
+        <div class="cards__row">
+          <div class="card card--margin" data-qa="card">
+            <img
+              class="card__product-img"
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            >
+
+            <h3 class="card__title">
+              APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+
+            <p class="card__product-code">
+              Product code: 195434
+            </p>
+
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__reviews">
+                Reviews: 5
+              </p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">
+                Price:
+              </p>
+
+              <p class="card__price-amount">
+                &dollar;2,199
+              </p>
+            </div>
+
+            <a href="#"
+              class="card__link"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+
+          <div class="card card--margin">
+            <img
+              class="card__product-img"
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            >
```

Comment:
```suggestion
          <div class="card">
            <img
              class="card__img"
              src="./images/imac.jpeg"
              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
            >
```
no need create extra classes
`card--margin` - not the best name